test/file_system_entity_extensions_test.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:io' as io;
```

Comment:
Let's use universal_io

Add: https://pub.dev/packages/universal_io
Then replace all `import 'dart:io'` with `import 'package:universal_io/io.dart'`

Reply:
Was only used in one place, in a test, for an exception type, that's exports to the `dart:io` value....
but it's good practice to not import `dart:io` so I'll do it :-)